src/marketing/event-ticker.md

```diff
@@ -5,10 +5,10 @@ title: Event Ticker
 
 The ticker block displays a countdown ticker for open events, with the start and end date for upcoming events. If an event has closed, the ticker shows the starting and ending dates.
 
-![]({% link images/images-ee/storefront-event-ticker-carousel.png %}){: .zoom}
-*Catalog Event Carousel*
+![Example storefront - event carousel]({% link images/images-ee/storefront-event-ticker-carousel.png %}){: .zoom}
+_Catalog Event Carousel_
 
 If the Category Page ticker is enabled for an event, the ticker block appears at the top of the category listing. If the Product Page ticker is enabled, the ticker block also appears at the top of the product page of any product that is associated with the category. .
```

Comment:
where do i enable the ticker? i don't see it in configuration, create, or widget. looks like it might be part of the widget layout, but there is no reference to it in the widget config.

Reply:
Good question. I think it is probably part of the layout for the carousel, and included with the Luma default setup But we should have something to explain that here and tell people how to create that in a custom layout if needed..... another GitHub issue!